ui/src/components/WMWorkersTable/index.jsx

```diff
@@ -0,0 +1,257 @@
+                  </Typography>
+                }
+              />
+              <ContentCopyIcon size={iconSize} />
+            </TableCellListItem>
+          </TableCell>
+        </CopyToClipboard>
+
+        <TableCell>
+          {latestTask ? (
+            <TableCellListItem
+              button
+              component={Link}
+              to={`/tasks/${latestTask.run.taskId}/runs/${
+                latestTask.run.runId
+              }`}>
+              <ListItemText
+                disableTypography
+                primary={<Typography>{latestTask.run.taskId}</Typography>}
+              />
+              <LinkIcon size={iconSize} />
+            </TableCellListItem>
+          ) : (
+            <Typography>n/a</Typography>
+          )}
+        </TableCell>
+
+        {latestTask ? (
+          <CopyToClipboard
+            title={`${latestTask.run.started} (Copy)`}
+            text={latestTask.run.started}>
+            <TableCell>
+              <TableCellListItem button>
+                <ListItemText
+                  disableTypography
+                  primary={
+                    <Typography>
+                      <DateDistance from={latestTask.run.started} />
+                    </Typography>
+                  }
+                />
+                <ContentCopyIcon size={iconSize} />
+              </TableCellListItem>
+            </TableCell>
+          </CopyToClipboard>
+        ) : (
+          <TableCell>
+            <Typography>n/a</Typography>
+          </TableCell>
+        )}
+
+        {latestTask ? (
+          <CopyToClipboard
+            title={`${latestTask.run.resolved} (Copy)`}
+            text={latestTask.run.resolved}>
+            <TableCell>
+              <TableCellListItem button>
+                <ListItemText
+                  disableTypography
+                  primary={
+                    <Typography>
+                      <DateDistance from={latestTask.run.resolved} />
+                    </Typography>
+                  }
+                />
+                <ContentCopyIcon size={iconSize} />
+              </TableCellListItem>
+            </TableCell>
+          </CopyToClipboard>
+        ) : (
+          <TableCell>
+            <Typography>n/a</Typography>
+          </TableCell>
+        )}
+
+        <TableCell>
+          <TableCellListItem
+            button
+            component={Link}
+            to={`${path}/worker-types/${workerType}/workers/${workerGroup}/${workerId}/recent-errors`}>
+            <ListItemText
+              disableTypography
+              primary={<Typography>Click to see errors</Typography>}
+            />
+            <AlertIcon size={iconSize} />
+          </TableCellListItem>
+        </TableCell>
+
+        <TableCell>
+          <TableCellListItem
+            button
+            component={Link}
+            to={`${path}/worker-types/${workerType}/workers/${workerGroup}/${workerId}/resources`}>
```

Comment:
I see a similar behavior to the previous comment re: "Click to see errors".

Reply:
yeah, same thing. I'll remove these links altogether for now.